linera-client/src/unit_tests/chain_listener.rs

```diff
@@ -161,7 +161,7 @@ async fn test_chain_listener() -> anyhow::Result<()> {
             delivery,
             false,
             [chain_id0],
-            "Client node",
+            format!("Client node for {:.8}", chain_id0),
```

Comment:
It is a minor point, but I find it a little odd that the `Client` has a `name` associated with its constructor. Though of course, this is not part of this PR.

Reply:
Yes, like @ma2bd said above: Let's name the _wallets_ instead. I created https://github.com/linera-io/linera-protocol/issues/2551 for that.